sample-code/src/orchestration.ts

```diff
@@ -354,10 +356,14 @@ export async function orchestrationChatCompletionImage(): Promise<OrchestrationR
     }
   });
 
+  const imageFilePath = join(__dirname, 'src', 'media', 'sample-image.png');
+  const mimeType = lookup(extname(imageFilePath).toLowerCase());
```

Comment:
[pp] Computing the mime type is maybe a bit overkill, since we anyway hardcode png here in the filename. Thus might Asseln hard code the mime type ;)

Reply:
I agree that hardcoding the mime type here is probably enough, but please keep it separate from the template string, so our users still know which part to replace, in case their image is of a different type.